plugins/by-name/blink-emoji/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "blink-emoji";
+  package = "blink-emoji";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  description = ''
+    This plugin should be configured through blink-cmp's `sources.providers` settings.
+
+    For example:
+
+    ```nix
+    plugins.blink-cmp = {
+      enable = true;
+      settings.sources.providers = {
+        emoji = {
+          module = "blink-emoji";
+          name = "Emoji";
+          score_offset = 15;
+          # Optional configurations
+          opts = {
+            insert = true;
+          };
+        };
+      };
+    };
+    ```
+
+    And then you can add it to blink-cmp's `default` sources:
```

Comment:
Q: is "`default` source" or "`default` sources" more appropriate here?

Reply:
Or maybe we just say "`sources.default` setting"?

Reply:
Probably best to just pass the option name because both wording is weird imo